<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addContactManual</class>
 <widget class="QMainWindow" name="addContactManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>200</width>
      <height>338</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Name_Lab">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Name_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Age_Lab">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Age_Spin"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="Location_Lab">
         <property name="text">
          <string>Location</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Location_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="Gender_Lab">
         <property name="text">
          <string>Gender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Gender_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="Ethnicity_Lab">
         <property name="text">
          <string>Ethnicity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Ethnicity_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="Education_Lab">
         <property name="text">
          <string>Education</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Education_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="Occupation_Lab">
         <property name="text">
          <string>Occupation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Occupation_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="Email_Lab">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Email_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="Phone_Num_Lab">
         <property name="text">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Phone_Num_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="Add_Info_Lab">
         <property name="text">
          <string>Additional Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Add_Info_Line"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="Add_Contact_But">
       <property name="text">
        <string>Add Contact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Return_To_CM_But">
       <property name="text">
        <string>Return to Contact Manager</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
